alerter.c

```diff
@@ -1,19 +1,50 @@
 #include <stdio.h>
 #include <assert.h>
 
-int alertFailureCount = 0;
+extern void alertInCelcius(float farenheit);
+extern int GetalertFailureCount(void);
+extern int networkAlertStub(float celcius);
+
+int networkAlert(float celcius)

```

Comment:
Keeping test code separate from production code helps in many ways, long term maintainability and test code can be excluded easily during shipment of software to field.

Reply:
this function was introduced with an intention to separate the network-alerter functionality with other applicaiton code and test code. With an idea to easily integrate the network alerter functionality. Can you please elaborate which part of the code belongs to test code. @prithamvshetty

Reply:
Networkalertstub is a test code. For this session as you are not required to change production code, you can still rearrange them. Maybe I mistook the code rearrangement as production code modification 👍 As of now concentrate on strengthening test cases to cover all scenarios 👍

Reply:
thanks